src/Microsoft.AspNetCore.SignalR/HubConnectionList.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubConnectionList : IReadOnlyCollection<HubConnectionContext>
```

Comment:
It's just not important enough at the moment. ConnectionList isn't really that fundamental, it's there for convenience.